scss/civicrm/common/_buttons.scss

```diff
@@ -382,7 +383,6 @@ button {
   border-radius: $border-radius-base;
   color: $crm-white;
   font-family: $font-family-base;
-  height: 36px;
```

Comment:
Now that Vanilla was working fine with this change. Can you make sure that any page is not left. 
If backstop is not loading page than check that page manually?
This is very generic and I want be absolutely sure that, we don't break anything due to this

Reply:
Some backstop pages did not load, I tested them manually. A couple of pages (such as pledge dashboard) do not open for me, but this change does only change buttons and I tested enough buttons on other pages - they look fine.

Reply:
OK thanks